Wikipedia/UI-V5/WMFHomeSectionSchema.m

```diff
@@ -251,7 +259,28 @@ - (WMFHomeSection*)mainPageSection {
     return main;
 }
 
-- (WMFHomeSection*)continueReadingSection {
+- (nullable WMFHomeSection*)featuredArticleSection {
+    //Never show if we are not in english
+    if (![self.site.language isEqualToString:@"en"] && ![self.site.language isEqualToString:@"en-US"]) {
```

Comment:
@montehurd i guess in this case we only want en.wikipedia.org, so checking for "en" should be better. there is no "en-us.wikipedia.org"